### DISCUSSION TITLE
Multiple MongoDB Connections

### DISCUSSION THREAD

**Discussion by Lishenga** [mikro-orm/mikro-orm#2375] [Q&A]
Hello,

You're truly doing an amazing job with MikroOrm, it's super amazing, i had a question, does MikroOrm support multiple database connections to a Mongo Database ?

Thank you

**Comment by B4nan** [comment]
You can have multiple ORM instances for such use case. The `RequestContext` helper also supports such use case, see the tests for more info:

https://github.com/mikro-orm/mikro-orm/blob/master/tests/RequestContext.test.ts#L67
